feat: allow not_null value for the is operator (#3818)
PostgREST/postgrest: CHANGELOG.md

 - #3560, Log resolved host in "Listening on ..." messages - @develop7
 - #3727, Log maximum pool size - @steve-chavez
 - #1536, Add string comparison feature for jwt-role-claim-key - @taimoorzaeem

>  - #3747, Add `not_null` filter value to pgrst query - @taimoorzaeem

Reviewer: ```suggestion
 - #3747, Allow `not_null` value for the `is` operator - @taimoorzaeem
```
